config/environments/test.rb

```diff
@@ -49,4 +49,8 @@
 
   # Use separate cache stores for parallel_tests
   config.cache_store = :file_store, Rails.root.join("tmp", "cache", "paralleltests#{ENV['TEST_ENV_NUMBER']}")
+
+  # Disables automatic creation of the test DB so that it has to be explicitly
+  # generated before running tests.
+  config.active_record.maintain_test_schema = false
```

Comment:
Right, I could 'enhance' the test task here. Or I could simply call `Rake::Task['jenkins:db']` at the beginning of test_helper if the db hasn't been initialized yet. Otherwise the developer would need to run db:migrate before test on a new installation.

The core of the issue here is validate_inclusion_in_families calling `attribute_names` before the db is populated. I don't think that should hold up the PR though, it can be fixed after we have Rails 4.